src/pages/home/sidebar/optionPropTypes.js

```diff
@@ -4,8 +4,8 @@ export default PropTypes.shape({
     // The full name of the user if available, otherwise the login (email/phone number) of the user
     text: PropTypes.string.isRequired,
 
-    // The login of the user, or the name of the chat room
-    alternateText: PropTypes.string.isRequired,
+    // The login of the user, or the last message of the chat room
+    lastMessageText: PropTypes.string.isRequired,
```

Comment:
Comment is out of dat so that can be changed. But we should use something more generic and return to using `alternateText`. `lastMessageText` is too specific.